src/sequences/SequenceRandom.ts

```diff
@@ -0,0 +1,77 @@
+import { ValidationStatus } from '@/shared/ValidationStatus';
+import { SequenceParamsSchema, SequenceExportModule,
+         SequenceExportKind } from './SequenceInterface';
+import { SequenceCached } from './SequenceCached';
+
+/**
+ *
+ * @class SequenceClassRandom
+ * Creates a sequence of random integers in a specified range.
+ *
+ */
+class SequenceRandom extends SequenceCached {
+	name = "Random Integers in Range";
+	description = "A sequence of integers chosen indepenently uniformly from n to m inclusive.";
+	params: SequenceParamsSchema[] = [new SequenceParamsSchema(
+		'min',
+		'number',
+		'Minimum value attainable',
+		false,
```

Comment:
This is the argument that specifies whether the parameter `min` is required. Looking at validate() below, clearly it is required. Hence this should be true, and the same goes for `max` below. I will also file an issue that if a parameter has been marked as required, the Sequence writer should not also have to check it manually in validate(). (Right now this parameter and the code in validate() are partially redundant.)